src/main/java/co/inventrosoft/hibernate_task/console/ConsoleParser.java

```diff
@@ -5,26 +5,25 @@
 import co.inventrosoft.hibernate_task.model.Match;
 import co.inventrosoft.hibernate_task.model.Team;
 
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
 
 import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.InputStreamReader;
 import java.util.ArrayList;
 
 import java.util.List;
 
 
 @Component
+@RequiredArgsConstructor
 public class ConsoleParser {
     private final BufferedReader reader;
```

Comment:
It is not used here.
If you have public close method for this object it means that you have control over it on upper level.